keras_hub/src/models/yolo_v8/yolo_v8_detector_test.py

```diff
@@ -0,0 +1,318 @@
+import os
+
```

Comment:
add generic task test - self.run_task_test - example : https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/bert/bert_text_classifier_test.py#L41

Reply:
you will need to add preprocessor flow - example follow resnet - https://github.com/keras-team/keras-hub/tree/master/keras_hub/src/models/resnet